src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import java.nio.file.Path;
+import java.util.Scanner;
+
+import duke.commands.Command;
+import duke.commands.CommandType;
+import duke.exceptions.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Duke is a chatbot that helps you keep track of your tasks.
+ * In this version, Duke is renamed to LilBro.
+ */
+public class Duke {
+
+    /** The task list that is used to store the user's tasks. */
+    private final TaskList taskList = new TaskList();
+
+    /** The storage that is used to save and load the user's tasks. */
+    private final Storage storage;
+
+    /**
+     * Constructs a new Duke object.
+     *
+     * @param path The path to the save file.
+     */
+    public Duke(String path) {
+        Ui.greet();
+        Ui.println("Checking for a save file...");
+        String projectRoot = System.getProperty("user.dir");
+        String fullPath = Path.of(projectRoot, path).toString();
+        this.storage = new Storage(fullPath);
```

Comment:
Adding on to the previous comment, perhaps you could possibly do this since you are already using Paths:
```java
Path fullPath= Paths.get(".").resolve(inputPath);
```
`Paths.get(".")` gets the current working directory and you can add on the user input using `resolve(inputPath)`

You can take a look [here](https://docs.oracle.com/javase/tutorial/essential/io/pathOps.html#create) for Path operations :)